setup.cfg

```diff
@@ -12,6 +12,7 @@ description = fMRIPrep is a robust and easy-to-use pipeline for preprocessing of
 license = 3-clause BSD
 long_description = file:long_description.rst
 long_description_content_type = text/x-rst; charset=UTF-8
+name= fmriprep
```

Comment:
this, the `setup_requires`, and the changes to `setup.py` are opportunistic, at some point, I needed to trigger a build and it seemed a nice cleanup pretty harmless.